roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -46,10 +46,24 @@
   register: common_version
   failed_when: false
   changed_when: false
-  when: not openshift.common.is_atomic | bool
+  when: not openshift.common.is_containerized | bool
+
+- set_fact:
+    l_common_version: "{{ openshift.common.image_tag | oo_image_tag_to_rpm_version | default('0.0', True) }}"
+  when: openshift.common.is_containerized | bool
+
+- set_fact:
+    l_common_version: "{{ common_version.stdout | default('0.0', True) }}"
+  when: not openshift.common.is_containerized | bool
```

Comment:
I probably have an irrational hate of set_fact...  I can't think of a way to put this into a vars file though, with the use of common_version.stdout though...

it could be limited down to one set_fact if you do the following though:
```
l_common_version: "{{ ( openshift.common.image_tag | oo_image_tag_to_rpm_version if openshift.common.is_containerized | bool else common_version.stdout) | default('0.0', True) }}"